src/main/resources/assets/additionalguns/models/item/vintorez.json

```
@@ -1,320 +1,271 @@
 {
-	"credit": "Made with Blockbench by ThatAvocado_Boi",
+	"credit": "Made with Blockbench",
 	"texture_size": [32, 32],
 	"textures": {
-		"3": "additionalguns:item/vintorez"
+		"1": "additionalguns:item/anvil",
+		"2": "additionalguns:item/universal3",
+		"6": "additionalguns:item/stripped_oak_log",
```

Same goes for this one, you'll get the idea :)
(Please also remove the textures from your PR)
```suggestion
		"6": "minecraft:item/stripped_oak_log",
```